cmd/osbuild-worker-executor/handler_build.go

```diff
@@ -84,6 +84,7 @@ func runOsbuild(logger *logrus.Logger, buildDir string, control *controlJSON, ou
 	// #nosec G204
 	cmd = exec.Command(
 		"tar",
+		"--exclude=output/output.tar",
```

Comment:
Nitpick: it would be IMO better to not hardcode the output directory and join `outputDir` with `"output.tar"` as it is done below.